CONTRIBUTING.md

```diff
@@ -94,3 +94,48 @@ Then you should be able build it via Maven using the goals `clean package`.
 
 If you have any further questions, then please join our [Discord Support Server](https://discord.gg/slimefun) and ask your questions in the `#programming-help` channel.<br>
 **Note that we will not accept any bug reports from custom-compiled versions of Slimefun**.
+
+## :black_nib: Code Style guidelines
+The general gist when it comes to code style: **Try to be consistent!**.<br>
+Try to stay inline with the code that surrounds you, having an entire package or even a single file that's filled with plenty of different and inconsistent code styles is just hard to read or maintain. That's why we wanna make sure everyone follows these simple principles:
+
+1. Imports
+* Don't use wildcard (`*`) imports!
+* Don't import unused classes!
+* Don't use static imports!
+2. Annotations
+* Methods and parameters should be annotated with `@Nullable` or `@Nonnull`!
```

Comment:
Should also tell from which api/package.

Reply:
Yes!